### PR TITLE
A few SQL changes + typo fix

### PR DESCRIPTION
**Pull request by xykev**

### CODE REVIEW

File: mysql_example/mysql_example.py [hunk 1]
```diff
@@ -276,7 +277,8 @@ def dbWriter(sql, rows) :
 c.execute("CREATE TABLE plural_block "
           "(SELECT block_id, donor_id "
           " FROM blocking_map INNER JOIN plural_key "
-          " USING (block_key))")
+          " USING (block_key)"
+          " GROUP BY block_key, donor_id)")
```
Comment (fgregg) [inline mysql_example/mysql_example.py]: Have you found instances where the one donor_id is covered multiple times by the same block_key?

File: mysql_example/mysql_example.py [hunk 2]
```diff
@@ -371,7 +373,7 @@ def candidates_gen(result_set) :
           "(donor_id INTEGER, canon_id INTEGER, "
           " cluster_score FLOAT, PRIMARY KEY(donor_id))")
 
-for cluster, score in clustered_dupes :
+for cluster, scores in clustered_dupes :
```
Comment (fgregg) [inline mysql_example/mysql_example.py]: Good catch?

File: mysql_example/mysql_example.py [hunk 3]
```diff
@@ -267,7 +267,8 @@ def dbWriter(sql, rows) :
           " block_id INTEGER UNSIGNED AUTO_INCREMENT, "
           " PRIMARY KEY (block_id)) " 
           "(SELECT block_key FROM blocking_map "
-          " GROUP BY block_key HAVING COUNT(*) > 1)")
+          " GROUP BY block_key HAVING COUNT(*) > 1)"
+          "CHARACTER SET utf8 COLLATE utf8_unicode_ci")
```
Comment (fgregg) [inline mysql_example/mysql_example.py]: I would prefer to set CHARACTER SET and COLLATION at the connection level.

File: mysql_example/mysql_example.py (hunk 1 above)
Comment (xykev) [inline mysql_example/mysql_example.py]: It has happened for me, although it is admittedly rather strange.

Comment (fgregg) [inline mysql_example/mysql_example.py]: This really should not be happening. Would you mind seeing if you can reproduce and posting the predicates that dedupe had learned on that run?

Comment (xykev) [inline mysql_example/mysql_example.py]: Oops, my mistake. I had previously edited some code elsewhere in the program that caused some errors. This change should be ignored.

**Comment by fgregg** [comment]
Can you address the issues I raised, and commit the new changes. (It will automatically update this PR).

**Pull request merged**